cloudinit/sources/DataSourceOVF.py

```diff
@@ -140,6 +141,10 @@ def _get_data(self):
             try:
                 cf = ConfigFile(vmwareImcConfigFilePath)
                 self._vmware_cust_conf = Config(cf)
+                post_gc_status = self._vmware_cust_conf.post_gc_status
+                if post_gc_status:
+                    set_gc_status("Started")
```

Comment:
Thanks OddBloke. If we move the check of post_gc_status into set_gc_status, then we need to pass self._vmware_cust_conf  into set_gc_status() because set_gc_status is not a function of DataSourceOVF.py.

Reply:
Yep, I think that's fine to achieve our goal here.